cypress/integration/login.js

```diff
@@ -23,4 +23,48 @@ describe("Login functionality", function() {
       .should("have.length", 2);
     cy.get(".calendar--today").contains(`${new Date().getDate()}`);
   });
+
+  const rolesData = [
+    {
+      name: "User",
+      tabs: 1
+    },
+    {
+      name: "Team Lead",
+      tabs: 2
+    },
+    {
+      name: "Manager",
+      tabs: 5
+    },
+    {
+      name: "Accountant",
+      tabs: 5
+    },
+    {
+      name: "Admin",
+      tabs: 6
+    }
+  ];
```

Comment:
You could try to use a different data structure for a better readability, e.g.:
const myObject = {
  first:  { key:  'thing', key2: 2 },
  second: { key3: 'otherThing' }, 
  third:  { key4: 'my string' }
};